dl/helpers/utils.py

```diff
@@ -106,6 +106,22 @@ def convert(inp,outfmt='pandas',verbose=False,**kwargs):
        df = convert(inp,'pandas',na_values='Infinity') # na_values is a kwarg; adds 'Infinity' to list of values converter to np.inf
 
     """
+    index = inp.find('\n')
```

Comment:
This is good Wendy.  I might had a comment as to what we are doing in this "paragraph" of code and why.

Reply:
Comments are added. Thanks for checking!